org.mixedrealitytoolkit.spatialmanipulation/Solvers/RadialViewDisplayInitializeHelper.cs

```diff
@@ -0,0 +1,27 @@
+using UnityEngine;
+
+namespace MixedReality.Toolkit.SpatialManipulation
+{
+    /// <summary>
+    /// Makes a game object appear having a RadialView component in the center of the view,
+    /// at a distance equal to the average of the MinDistance and MaxDistance of the RadialView.
+    /// This prevents the object from 'racing into your view' and/or appear slanted
+    /// </summary>
+    [RequireComponent(typeof(RadialView))]
+    public class RadialViewDisplayInitializeHelper : MonoBehaviour
+    {
+        private RadialView radialView;
+
+        protected void Awake()
+        {
+            radialView = GetComponent<RadialView>();
+        }
+
+        private void OnEnable()
```

Comment:
As a best practive we are requiring that MonoBehaviour event functions be at least "protected", if the class is not sealed.

This requirement is meant to prevent subclasses from accidentally hiding the base's implementation of event functions (Awake, OnEnable, ect.).  That is, if the base event functions are private and the subclass implemented its own version, the compiler will happily allow this (no warning, no error).   This would then hide the base implementation. 

If the base event functions are protected, the compiler will at least provide a warning message.

So please make  `OnEnable` projected.